Unity-Technologies/Graphics
# Srp workflows/align overdraw workflow

### Purpose of this PR
This align the overdraw debugging workflows for both URP and HDRP. (Initiated by Fogbugz [1365666 ](https://fogbugz.unity3d.com/f/cases/1365666/))

HDRP:
- Updated the color feedback
<img width="374" alt="Unity_pn9UtJDGKE" src="https://user-images.githubusercontent.com/55485372/149916000-4acec676-b7b8-48e4-8c94-79840a6a9720.png">

URP:
- Added Overdraw Debug Mode and Overdraw Max Pixel Cost
![image](https://user-images.githubusercontent.com/55485372/135846932-8e89d6f4-054d-4eda-93a0-65b312386c5e.png)
- Overdraw Debug Mode All
<img width="374" alt="Unity_OCZdUodJ0d" src="https://user-images.githubusercontent.com/55485372/149916267-f44ea927-7e9f-493d-b5f5-4e16a49118c3.png">
- Overdraw Debug Mode Transparent Only
<img width="374" alt="Unity_kjIFla6isC" src="https://user-images.githubusercontent.com/55485372/149916090-f07add44-585d-495c-9808-f8be01ff066f.png">
- Overdraw Debug Mode Opaque Only
<img width="374" alt="Unity_GlDR0TOnzz" src="https://user-images.githubusercontent.com/55485372/149916285-5d7b13ed-df3b-459b-9752-01753b929a5b.png">

---
### Testing status
Manual Test:
- Created a set of transparent and opaque cube overlapping each other, one per renderer (total of 4 sets)
- Activated all overdraw mode, one by one to make sure the color feedback is correct

---
### Comments to reviewers

HDRP: I updated the debug shader to use the Shader Core Library for the feedback color. The change is minimal here.

URP: I added a specific code to render the overdraw debug mode in several place. Can you check that it fit appropriately in URP?
